extensions/window/filter.lua

```diff
@@ -562,7 +564,7 @@ function WF:setAppFilter(appname,ft,batch)
           logs=sformat('%s%s={%s}, ',logs,k,first)
         else logs=sformat('%s%s=%s, ',logs,k,v) end
         filter.allowRoles=r
-      elseif k=='visible' or k=='fullscreen' or k=='focused' or k=='currentSpace' or k=='activeApplication' then
+      elseif k=='visible' or k=='fullscreen' or k=='focused' or k=='currentSpace' or k=='activeApplication' or 'hasTitlebar' then
```

Comment:
shouldn't that be `or k == 'hasTitlebar'`? As is, the line will always evaluate to true because a string constant is "true" in lua.